### ISSUE TITLE
GLES2/WebGL1 Renderbuffer creation failure

### ISSUE THREAD

**Issue by muiroc** [godotengine/godot#22629]
**Godot version:**
4d693fc3f0287df6cefbc0714fab28b6f825fcb2

**OS/device including version:**
Windows 10, Firefox 62.0.2, Chrome 69.0.3497.100 through default Angle backend (dx11 on HD4000)

**Issue description:**
Getting a black screen with both browsers with the error:
drivers\gles2\rasterizer_storage_gles2.cpp:3948:_render_target_allocate() - Could not create framebuffer!!

index.js:4 WebGL: INVALID_ENUM: renderbufferStorage: invalid internalformat

Looks like glRenderbufferStorage doesn't accept the 0x81A6 enumeration, temporarily forcing it to DEPTH_COMPONENT16 avoids the error. (also failed when passed GL_DEPTH_COMPONENT unsized enum)

Looks like there are no exts for DEPTH_COMPONENT24, only available in WebGL2?

To quickly test this without a full rebuild, you can modify the emitted index.js:
```javascript
function _glRenderbufferStorage(x0, x1, x2, x3) {
	x1 = GLctx.DEPTH_COMPONENT16
	GLctx["renderbufferStorage"](x0, x1, x2, x3)
}
```
This way it succeeds and shows something on screen

NOTE if you modified index.js and need to reload the page in chrome, be sure to do a full reload (hold the reload button, last popup option)

**Comment by muiroc** [comment]
@LinuxUserGD 
It was similar to this: [cube.zip](https://github.com/godotengine/godot/files/2568590/cube.zip), but also failed in a larger project

Interestingly I just re-exported and tried again in Firefox 63 chrome 70 and it didn't fail!
(the export template is kind of old)

There are some errors but looks unrelated (since not from glRenderbufferStorage?):
Error: WebGL warning: texImage2D: With format DEPTH_COMPONENT24, this function may only be called with target=TEXTURE_2D, data=null, and level=0.
WebGL warning: framebufferTexture2D: `level` must be 0. 

The background environment is visible but the cube is black

I didn't follow these things in a while, I will rebuild to latest master and try again

**Comment by muiroc** [comment]
fixed by #25222
